dandiapi/api/views/dashboard.py

```diff
@@ -82,6 +83,26 @@ def _users(self):
         )
 
 
+def mailchimp_csv_view(request: HttpRequest) -> HttpResponse:
+    """Generate a Mailchimp-compatible CSV file of all active users."""
+    # In production, there's a placeholder user with a blank email that we want
+    # to avoid.
+    users = User.objects.filter(is_active=True).exclude(email='')
+    data = users.values_list('email', 'first_name', 'last_name')
```

Comment:
Maybe use [.iterator](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#iterator).

Reply:
Done in e2808d15baf4643352924eaecffea9643a26d9af.